### PR TITLE
grpc: add more empty types

### PR DESCRIPTION
**Pull request by markya0616**

### CODE REVIEW

File: grpc/types.go [hunk 1]
```diff
@@ -97,6 +99,28 @@ func (m Method) ResponseName() string {
...
+			IsSlice: false,
+		},
+	},
+}
+
+var EmptyConstReq = Message{
+	Name: "EmptyConstReq",
+}
+
+var EmptyRsp = Message{
```
Comment (alanchchen) [inline grpc/types.go]: So, it's not really empty, right ?
How about using `TransactionResp` or `TxResp`?

File: grpc/types.go [hunk 2]
```diff
@@ -97,6 +99,28 @@ func (m Method) ResponseName() string {
 
 var EmptyReq = Message{
 	Name: "EmptyReq",
+	Args: []Argument{
```
Comment (alanchchen) [inline grpc/types.go]: I think we should give it a new name, try not to use 'empty'.

Comment (markya0616) [reply grpc/types.go]: OK. Rename to TransactionReq

File: grpc/types.go (hunk 1 above)
Comment (markya0616) [reply grpc/types.go]: OK. Rename to `Empty`.

**Review by alanchchen — APPROVED** [review]

**Pull request merged**